### ISSUE TITLE
NAT and UDP packets

### ISSUE THREAD

**Issue by slater0013** [martin-ger/esp32_nat_router#39]
Hello,

Trying to use esp32_nat_router with a big wifi antenna connected to a WROOM_32U to extend my Drone range.

Installation is ok, configuration is ok.

But my drone is sending the video stream as an UDP feed. And I suppose the NAT router does not know to which client send it !

Is there any way to specify a "DMZ" NATed client that would receive every incoming packets, including UDP packets that are not in the NAT table.

Hope It's clear ?!

Thanks,
GS.

**Comment by martin-ger** [comment]
No, there is no default client. You can define a portmap rule for any required UDP port.

**Comment by slater0013** [comment]
Ok ! Any Link to a documentation in order to achieve such rules ?
Thanks !

**Comment by slater0013** [comment]
Sorry for asking again, any hints how to add such port redirections ?

**Comment by martin-ger** [comment]
You have to modify the  code: add the call(s) to ip_portmap_add() after esp32_nat_router.c#L367

**Comment by skorokithakis** [comment]
A DMZ mode would be great, as I'm also running a single client behind this. I don't know how much work a fully-transparent "repeater" mode would be (ie it just extends the current LAN by transporting packets back and forth, without NAT), but that would be ideal.
